posts/2023-12-18-survey-launch.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Launching the 2023 State of Rust Survey"
+author: The Rust Survey Working Group
+description: "Share your feedback with the eighth edition of the State of Rust Survey"
+---
+
+It’s time for the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc)!
+
+Since 2016, the Rust Project has collected valuable information and feedback from the Rust programming language community through our annual [State of Rust Survey](https://www.surveyhero.com/c/4vxempzc). This tool allows us to more deeply understand how the Rust Project is performing, how we can better serve the global Rust community, and who our community is composed of.
+
+Like last year, the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc) will likely take you between 10 and 25 minutes, and responses are anonymous. We will accept submissions until Monday, January 15th, 2024. Trends and key insights will be shared on [blog.rust-lang.org](https://blog.rust-lang.org) as soon as possible in 2024.
+
+We invite you to take this year’s survey whether you have just begun using Rust, you consider yourself an intermediate to advanced user, or you have not yet used Rust but intend to one day. Your responses will help us improve Rust over time by shedding light on gaps to fill in the community and development priorities, and more.
+
+**Once again, we will be offering the State of Rust Survey in the following languages (if you speak multiple languages, please pick one on the main survey page):**
```

Comment:
Change to "Once again, we are offering the State of Rust Survey in the following languages (if you speak multiple languages, please pick one. Language options are available on the [main survey page](https://www.surveyhero.com/c/4vxempzc)"